pandas/core/frame.py

```diff
@@ -3204,7 +3204,10 @@ def reindexer(value):
             value = reindexer(value).T
 
         elif isinstance(value, ExtensionArray):
+            from pandas.core.series import _sanitize_index
             value = value.copy()
+            # Copy donesn't have any effect at the moment
```

Comment:
why are you adding this comment? this is on-purpose

Reply:
The comment isn't clear, sorry. It's to answer "why call `value.copy()`" and then pass it to a function that takes a `copy` parameter. That's because `__sanitize_index` doesn't copy an EA, even with `copy=True`.